xml/issue4389.xml

```diff
@@ -29,6 +29,13 @@ implementation to tell whether a class has member `begin/end` but the correspond
 call is ill-formed with C++20 core language rules, and such determination is critical for 
 eliminating the semantic differences between `ranges::for_each` and range-for.
 </p>
+
+<note>2025-10-23; Reflector pool; Status changed: New &rarr; SG9 and P2.</note>
+<p>
+This is certainly evoluationary question and should go to LEWG/SG9.
+It would disallow having unrelated begin/end members, whilke range
+interface is provided by hidden friends.
```

Comment:
```suggestion
It would disallow having unrelated begin/end members, where the range
interface is provided by hidden friends instead of those members.
```

I think this clarifies the comment.